core/server/src/main/java/alluxio/worker/netty/DataServerHandler.java

```diff
@@ -68,11 +50,9 @@
    * @param configuration Alluxio configuration
    */
   public DataServerHandler(final BlockWorker blockWorker, Configuration configuration) {
```

Comment:
I was thinking of making the `*DataServerHandlers` owned by `DataServerHandler`. Do you mean to have the `DataServer` implementation own all the handlers?